src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -180,18 +183,18 @@ public void validateIntegrationTestClassNames() throws Exception {
                         .that(matcher.matches())
                         .isTrue();
 
-                final String check = matcher.group(1);
-                assertWithMessage("Unknown check '" + check + "' in test file: " + filename)
+                final String test = matcher.group(1);
+                assertWithMessage("Unknown test '" + test + "' in test file: " + filename)
                         .that(SIMPLE_CHECK_NAMES)
-                        .contains(check);
+                        .contains(test);
 
                 assertWithMessage(
-                            "Check '" + check + "' is now compatible with SuppressionXpathFilter."
+                        "Check '" + test + "' is now compatible with SuppressionXpathFilter."
```

Comment:
please revert change in this file.
the only improvement we can do is to fix typo 'now' ----> not: 
`"Check '" + test + "' is not compatible with SuppressionXpathFilter."`